.github/dependabot.yml

```
@@ -6,6 +6,20 @@ updates:
     schedule:
       interval: daily
     open-pull-requests-limit: 15
+    labels:
+      - "dependency-upgrade"
+    commit-message:
+      prefix: "3.2.x"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
```

I think we could be more permissive now

```suggestion
        update-types: ["version-update:semver-major"]
```